inspirehep/dojson/hep/fields/bd1xx.py

```diff
@@ -59,11 +57,15 @@ def _get_affiliations(value):
             u_values = force_force_list(value.get('u'))
             z_values = force_force_list(value.get('z'))
 
-            for u_value, z_value in zip_longest(u_values, z_values):
-                result.append({
-                    'record': get_record_ref(z_value, 'institutions'),
-                    'value': u_value,
-                })
+            if len(u_values) == len(z_values):
+                for u_value, z_value in zip(u_values, z_values):
+                    result.append({
+                        'record': get_record_ref(z_value, 'institutions'),
+                        'value': u_value,
+                    })
+            else:
+                for u_value in u_values:
```

Comment:
Future @kaplun! Is the git commit message enough? https://github.com/inspirehep/inspire-next/pull/1460/commits/0cf0914772d730000d22014cb74d64128b2b4f2a : )